### ISSUE TITLE
Running in GitHub Actions?

### ISSUE THREAD

**Issue by atomiks** [gidztech/jest-puppeteer-docker#33]
I am getting `docker-compose: not found`.

PR available here: https://github.com/atomiks/tippyjs/pull/759/files which shows the actions I am using and the Jest configs. It works locally for me but not in GitHub Actions.

**Comment by Fiszcz** [comment]
Yeah,

Unfortunately I have the same situation.

I think that **docker-puppeteer** package should not use **docker-compose** only for one docker image. We could use simple ``docker`` commands.

**Comment by gidztech** [comment]
I just replaced my usage of Travis with Github Actions for this project and it's working fine for me out the box. `actions/setup-node` has Docker and Docker Compose included [among other tools](https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md).

I believe the issue is you are using `actions-yarn`, which is deprecated, and uses a more raw Ubuntu image without those tools. If you replace your workflow with `setup-node` you'll be all set.

As for whether Docker Compose is really needed in this project, perhaps not, but I made the assumption that CI environments would have it included because it's very common.
